src/main/io/beeper.c

```diff
@@ -403,6 +394,32 @@ void beeperUpdate(timeUs_t currentTimeUs)
 }
 
 /*
+ * Beeper motor/esc handler function to be called periodically in loop. 
+ */
+#ifdef USE_DSHOT
+void beeperMotorUpdate(timeUs_t currentTimeUs)
+{
+    static timeUs_t nextBeepTime = 0;
+    if (areMotorsRunning() || currentTimeUs < nextBeepTime) {
+        return;
+    }
+    
+    if (IS_RC_MODE_ACTIVE(BOXMOTORBEEPER)) {
+        nextBeepTime = currentTimeUs + 1000000;
```

Comment:
You are missing `pwmDisableMotors()` here.

Reply:
I'll get that in there.